code/modules/clothing/glasses/glasses.dm

```diff
@@ -117,6 +117,7 @@
 	icon_state = "nvgmeson"
 	origin_tech = "magnets=4;engineering=5;plasmatech=4"
 	see_in_dark = 8
+	prescription_upgradeable = TRUE
```

Comment:
```suggestion
	prescription_upgradable = TRUE
```
This is why you should at least try to compile your changes.

Reply:
sorry, damn.